Cargo.toml

```diff
@@ -1,87 +1,87 @@
 [package]
-name = "compio"
-version = "0.3.0"
-edition = "2021"
-authors = ["Berrysoft <Strawberry_Str@hotmail.com>"]
-readme = "README.md"
-license = "MIT"
+name        = "compio"
```

Comment:
Which formatter do you use...?

Reply:
Uhhh some vscode formatter w/ custom config i dont really remember. Ill change the format back.